dashboard/app/helpers/application_helper.rb

```diff
@@ -30,11 +30,6 @@ def ago(from_time)
     "#{s} ago"
   end
 
-  def format_xml(xml)
-    doc = Nokogiri::XML(xml)
-    doc.to_xhtml
-  end
```

Comment:
Yep!  Noted in this commit: https://github.com/code-dot-org/code-dot-org/pull/15034/commits/f839170537b6b79cb0ac3ce6ee3890f266b3c7f7